packages/blade/src/components/Avatar/Avatar.web.tsx

```diff
@@ -31,6 +37,9 @@ const _Avatar: React.ForwardRefRenderFunction<BladeElementRef, AvatarProps> = (
     href,
     target,
     rel,
+    isSelected,
+    bottomAddon: BottomAddon,
+    topAddon: TopAddon,
```

Comment:
```suggestion
    topAddon,
```

Thought this is react component